src/App.tsx

```diff
@@ -12,161 +19,53 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={
+              ({ isActive }) => classNames('navbar-item',
+                {
+                  'has-background-grey-lighter': isActive,
+                })
+            }
+            to="/"
+          >
             Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+          </NavLink>
+
+          <NavLink
+            className={
+              ({ isActive }) => classNames('navbar-item',
+                {
+                  'has-background-grey-lighter': isActive,
+                })
+            }
+            to="/people"
           >
             People
-          </a>
+          </NavLink>
```

Comment:
You can also create a separated component for avoiding code duplication by custom NavLink.